services/motion/builtin/builtin_test.go

```diff
@@ -565,8 +315,6 @@ func TestMoveOnMapPlans(t *testing.T) {
 	})
 
 	t.Run("check that position-only mode executes", func(t *testing.T) {
-		// TODO(RSDK-5758): unskip this
```

Comment:
[RSDK-5758](https://viam.atlassian.net/browse/RSDK-5758) is merged but this test fails.
```
logger.Infof("endPos.Pose(): %v", spatialmath.PoseToProtobuf(endPos.Pose()))
logger.Infof("goalInBaseFrame: %v", spatialmath.PoseToProtobuf(goalInBaseFrame))
```
gives back
```
2024-01-08T20:12:58.292Z	INFO		builtin/builtin_test.go:334	endPos.Pose(): x:1319.994856119947 y:-0.002638248908500054 o_z:1.0000000000000002 theta:179.99935774587044

2024-01-08T20:12:58.292Z	INFO		builtin/builtin_test.go:335	goalInBaseFrame: x:1320 o_z:1 theta:180
```

However, when running locally this does not happen.
@raybjork any ideas?

Reply:
Without a larger investigation I don't know.  This PR is big enough as is so my preference is to make a new ticket to unskip this

Reply:
https://viam.atlassian.net/browse/RSDK-6271